src/TinyMDE.js

```diff
@@ -54,10 +54,11 @@ class Editor {
     this.createEditorElement(element);
     // TODO Placeholder for empty content
```

Comment:
I thought maybe this TODO is speaking to the same thing? if so, shall I remove it in scope of this?

Reply:
yeah that's probably right (tbh I don't fully remember but it sounds like it)